logstash/datadog_checks/logstash/logstash.py

```diff
@@ -220,24 +233,25 @@ def check(self, instance):
         )
 
     def _process_stats_data(self, data, stats_metrics, config):
-        for metric, desc in stats_metrics.iteritems():
+        for metric, desc in iteritems(stats_metrics):
             self._process_metric(data, metric, *desc, tags=config.tags)
 
-
-    def _process_pipeline_plugins_data(self, pipeline_plugins_data, pipeline_plugins_metrics, config, plugin_type, tag_name):
-        for plugin_data in pipeline_plugins_data.get(plugin_type,[]):
+    def _process_pipeline_plugins_data(self, pipeline_plugins_data,
+                                       pipeline_plugins_metrics, config,
+                                       plugin_type, tag_name):
```

Comment:
Let's do 
```python
    def _process_pipeline_plugins_data(
        self,
        pipeline_plugins_data,
	pipeline_plugins_metrics,
        config,
	plugin_type, 
        tag_name
):
```
It will be `black` friendly

Reply:
Done.